modules/slowzones/map/SlowSegmentLabel.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useMemo } from 'react';
+
+import type { SlowZoneResponse } from '../../../common/types/dataPoints';
+import type { LineMetadata } from '../../../common/types/lines';
+import { stringifyTime } from '../../../common/utils/time';
+
+import type { SlowZoneDirection, SlowZonesByDirection } from './segment';
+import { DIRECTIONS } from './segment';
+
+import styles from './SlowSegmentLabel.module.css';
+import DirectionIndicator from './DirectionIndicator';
+
+type SlowSegmentLabelProps = {
+  slowZonesByDirection: SlowZonesByDirection;
+  line: LineMetadata;
+  isHorizontal: boolean;
+};
+
+type SlowZoneLabelProps = {
+  direction: SlowZoneDirection;
+  slowZone: SlowZoneResponse;
+  color: string;
+  isHorizontal: boolean;
+};
+
+const SlowZoneLabel = (props: SlowZoneLabelProps) => {
```

Comment:
Same thing as I stated below
```suggestion
const SlowZoneLabel: React.FC<SlowZoneLabelProps> = (props) => {
```